### PR TITLE
docs: added _Accessing resources behind a VPN with Podman on Windows_

### LINKED ISSUE #1051: Podman Desktop WSL2 image does not appear to have internet connectivity (unlike Docker Desktop's WSL2 image connectivity which is fine)
### Bug description

podman pull docker.io/hello-world
Trying to pull docker.io/hello-world...
Error: initializing source docker://docker.io/hello-world: pinging container registry docker.io: Get "[https://docker.io/v2/":](https://docker.io/v2/%22:) dial tcp: lookup docker.io: Temporary failure in name resolution

### Operating system

Windows 11

### Version

0.10.0

### Steps to reproduce

podman pull docker.io/hello-world

### Relevant log output

```shell
Trying to pull docker.io/hello-world...
Error: initializing source docker://docker.io/hello-world: pinging container registry docker.io: Get "[https://docker.io/v2/":](https://docker.io/v2/%22:) dial tcp: lookup docker.io: Temporary failure in name resolution
```

### Additional context

docker pull docker.io/hello-world
Using default tag: latest
latest: Pulling from library/hello-world
2db29710123e: Pull complete
Digest: sha256:c77be1d3a47d0caf71a82dd893ee61ce01f32fc758031a6ec4cf1389248bb833
Status: Downloaded newer image for hello-world:latest
docker.io/library/hello-world:latest

### LINKED ISSUE #3556: Update the documentation on proxy and networking for windows, with the addition of the user networking mode
We have this documentation in progress:
https://github.com/containers/podman-desktop/pull/1541

Need to be updated with https://github.com/containers/podman/pull/18303 and https://github.com/containers/podman-desktop/issues/2865

This will also fix the missing documentation raised in this issue: https://github.com/containers/podman-desktop/issues/1051 

- [x] https://github.com/containers/podman-desktop/issues/3569

### LINKED ISSUE #3569: Document Accessing with Podman a network behind a user-controlled VPN on Windows
Document Accessing with Podman a network behind a user-controlled VPN on Windows

TLDR: In Podman Desktop 1.3.0 or greater, when initializing the Podman machine, select the **User mode networking** option. See: https://github.com/containers/podman-desktop/pull/3251, https://github.com/containers/podman-desktop/issues/2865 https://github.com/containers/podman-desktop/issues/3556 

Procedure initial draft:

# Accessing with Podman a network behind a user-controlled VPN on Windows

On Windows, when Podman requires to access resources that are behind a user-controlled VPN, enable the user mode networking in your Podman machine.

#### Prerequisites

- Windows host with updated WSL2.
- Podman 4.6.0 or greater.
- On the host, the user controls the VPN activation.

#### Procedure

1. To enable user mode networking, when creating the Podman machine, select the **User mode networking (traffic relayed by a user process).** option. 

2. SSH to the Podman machine

   ```
   $ podman machine ssh *<podman_machine_name>*
   ```

3. Create and edit the `/etc/resolv.conf/` file:

   ```
   nameserver 1.1.1.1
   nameserver 8.8.8.8
   ```

4. Create and edit the `/mnt/wsl/wsl.conf` file:
   
   ```
   [network]
   generateResolvConf = false
   ```

5. Exit the Podman machine SSH session.

   ```shell-session
   $ exit
   ```

6. Restart the Podman machine.

#### Verification

- You can access resources that are behind the VPN.

_Originally posted by @themr0c in https://github.com/containers/podman-desktop/discussions/3151#discussioncomment-6748041_

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

docs: added _Accessing resources behind a VPN with Podman on Windows_

### What issues does this PR fix or reference?

Fixes #3556. 
Fixes #3569.
Fixes #1051.
References #1051.

### CODE REVIEW

File: website/docs/proxy/using-a-vpn-on-windows.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   ```ini
+   [network]
+   generateResolvConf = false
+   ```
+
+5. Exit the Podman machine SSH session.
+
+   ```shell-session
+   $ exit
+   ```
```
Comment (eidottermihi) [inline website/docs/proxy/using-a-vpn-on-windows.md]: I'm not sure if steps 2 to 5 are always needed. In my case enabling user mode networking was enough to get network connectivity when the host is connected to a VPN (in my case Cisco AnyConnect).

In contrast, using 1.1.1.1 / 8.8.8.8 as DNS nameservers, you no longer can resolve "internal"/"corporate" Domains (which is why VPN is used in the first place).

Comment (themr0c) [reply website/docs/proxy/using-a-vpn-on-windows.md]: Indeed these steps look really weird to me. I would be happy to remove them entirely, or to narrow down the cases when they are required.

File: website/docs/proxy/using-a-vpn-on-windows.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+1. To enable user mode networking, when creating the Podman machine, select the **User mode networking (traffic relayed by a user process).** option.
+
+2. SSH to the Podman machine
+
+   ```shell-session
+   $ podman machine ssh *<podman_machine_name>*
+   ```
+
+3. Create and edit the `/etc/resolv.conf/` file:
```
Comment (ghost) [inline website/docs/proxy/using-a-vpn-on-windows.md]:
```suggestion
3. Create and edit the `/etc/resolv.conf` file:
```

File: website/docs/proxy/using-a-vpn-on-windows.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+   ```ini
+   [network]
+   generateResolvConf = false
+   ```
+
+5. Exit the Podman machine SSH session.
+
+   ```shell-session
+   $ exit
+   ```
```
Comment (themr0c) [inline website/docs/proxy/using-a-vpn-on-windows.md]:
```suggestion
```

File: website/docs/proxy/using-a-vpn-on-windows.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+On Windows, when Podman requires to access resources that are behind a user-controlled VPN, enable the user mode networking in your Podman machine.
+
+#### Prerequisites
+
+- Windows host with updated WSL2.
+- Podman 4.6.0 or greater.
+
+#### Procedure
+
+1. To enable user mode networking, when creating the Podman machine, select the **User mode networking (traffic relayed by a user process).** option.
```
Comment (themr0c) [inline website/docs/proxy/using-a-vpn-on-windows.md]:
```suggestion
- To enable user mode networking, when creating the Podman machine, select the **User mode networking (traffic relayed by a user process).** option.
```

**Review by eidottermihi — APPROVED** [review]
LGTM

File: website/docs/proxy/using-a-vpn-on-windows.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+sidebar_position: 5
+title: Podman behind a VPN on Windows
+description: Accessing resources behind a VPN with Podman on Windows
+tags: [podman, vpn, windows]
+keywords: [podman, vpn, windows]
+---
+
+# Accessing resources behind a VPN with Podman on Windows
+
+On Windows, when Podman requires to access resources that are behind a user-controlled VPN, enable the user mode networking in your Podman machine.
```
Comment (cdrage) [inline website/docs/proxy/using-a-vpn-on-windows.md]:
```suggestion
On Windows, if Podman needs to access resources behind a user-controlled VPN, enable user mode networking in your Podman machine.
```

File: website/docs/proxy/using-a-vpn-on-windows.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+#### Prerequisites
+
+- Windows host with updated WSL2.
+- Podman 4.6.0 or greater.
+
+#### Procedure
+
+- To enable user mode networking, when creating the Podman machine, select the **User mode networking (traffic relayed by a user process).** option.
```
Comment (cdrage) [inline website/docs/proxy/using-a-vpn-on-windows.md]:
```suggestion
- When creating the Podman machine, select the **User mode networking (traffic relayed by a user process)** option to enable user mode networking.
```

File: website/docs/proxy/using-a-vpn-on-windows.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+#### Verification
+
+- You can access resources that are behind the VPN.
```
Comment (cdrage) [inline website/docs/proxy/using-a-vpn-on-windows.md]:
```suggestion
- You can now access resources that are behind the VPN.
```

**Review by cdrage — APPROVED** [review]
